playground/next13-app-router/app/page.tsx

```diff
@@ -1,11 +1,9 @@
-import type {
-  ISbStoriesParams,
-  StoryblokClient,
-} from '@storyblok/react/rsc';
-import { getStoryblokApi } from '@/lib/storyblok';
 import {
-  StoryblokServerComponent,
+  type ISbStoriesParams,
+  type StoryblokClient,
+  StoryblokStory,
 } from '@storyblok/react/rsc';
+import { getStoryblokApi } from '@/lib/storyblok';
```

Comment:
Should be this import updated wince we are not using lib anymore?

Reply:
what do you mean? yes we are using it, line 25

Reply:
> we are not using lib anymore

@edodusi I just saw the playgrounds have their own `lib` directory internally, that's where my confusion was.